components/status/StatusCard.vue

```diff
@@ -95,12 +96,13 @@ const showReplyTo = $computed(() => !replyToMain && !directReply)
       <template v-if="status.inReplyToAccountId">
         <StatusReplyingTo
           v-if="showReplyTo"
-          ml-20px pt-1 pl-5
+          m="l-5" rtl-m="l-0 r-5"
+          p="t-1 l-5" rtl-p="l-0 r-5"
```

Comment:
From contributing guide:

- Do not use `left/right` padding and margin: for example `pl-1`. Use `padding-inline-start/end` instead. So `pl-1` should be `ps-1`, `pr-1` should be `pe-1`. Same rules applies for margin.
- Do not use `rtl-` classes, such as `rtl-left-0`.